build/ci/update_csv.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash -xe
+set +o pipefail
+
+sed -i "s+$current_operator_image+$wanted_operator_image+g" $csv_file
+sed -i 's/.*minKubeVersion.*/  minKubeVersion: 1.19.0/' $csv_file
```

Comment:
do we plan to maintain this from now on? and if not, what is our plan for the future?

Reply:
could we maybe add some skip logic if our `minKubeVersion` doesn't support a specific KinD version?

Reply:
changing the minKubeVersion version is like putting a skip on the K8S version because it always will be in the run with a minimum K8S version of 1.19, but we will always use a newer one and it won't care what the version

Reply:
then I think a skip is more appropriate, since there is no special significance for 1.19